docker/ubuntu1604/docker-entrypoint.sh

```diff
@@ -16,11 +16,3 @@ then
   chown ${UID} /home/jenkins
   [ -d /home/jenkins/.ssh ] && chown ${UID} /home/jenkins/.ssh
 fi
-
-# update alba & arakoon packages to latest/greatest
-
-##apt-get update -qq
-##apt-get install -qq -y alba arakoon
-
-# finally execute the command the user requested
-exec "$@"
```

Comment:
I have fully built the containers and I cannot see any further parameters are being passed on as supervisor is doing all the job. 
Apparently that extra exec is not doing much on that particular scenario.